### ISSUE TITLE
The waffle plots by journal are less useful with so many journals

### ISSUE THREAD

**Issue by rempsyc** [rempsyc/busara_dashboard#29]
The waffle plots by journal are less useful with so many journals. We might have to find alternative plotting strategies or be more selective for these waffe plots. Example:

<img width="561" alt="image" src="https://github.com/rempsyc/busara_dashboard/assets/13123390/a75878a5-96b0-4ebf-8344-a8b15a1534e8">

**Comment by psforscher** [comment]
Yes, agreed. I'm not sure what the options are but with a large number of journals, dynamic waffle plots (where you type in your journal of choice) make the most sense to me

**Comment by rempsyc** [comment]
Could be addressed eventually in #10
